js/packages/openinference-core/package.json

```diff
@@ -34,10 +34,8 @@
   },
   "dependencies": {
     "@arizeai/openinference-semantic-conventions": "workspace:*",
-    "@opentelemetry/core": "^1.25.1"
-  },
-  "peerDependencies": {
-    "@opentelemetry/api": ">=1.0.0 <1.9.0"
+    "@opentelemetry/core": "^1.25.1",
+    "@opentelemetry/api": "^1.9.0"
```

Comment:
this is a direct dep i think, also we need 1.9 since other packages are using 1.9 and the span interface has changed